### PR TITLE
chore: Bump Helm charts

### PR DESCRIPTION
**Pull request by Techassi**
Part of https://github.com/stackabletech/issues/issues/658

### CODE REVIEW

**Review by sbernauer — APPROVED** [review]
Changes LGTM. I leave it to you if you want to test the changes now or later

**Comment by Techassi** [comment]
Alright, we will test the changes after merging it as part of the scheduled testing in https://github.com/stackabletech/issues/issues/658.

**Pull request merged**
